lessons/Early-Math/translating/langs/en-us/pages/practice-translating.adoc

```diff
@@ -0,0 +1,25 @@
+= Translating
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+
+Each row represents a single arithmetic expression, written in three different forms. Fill in the empty spaces so that all three forms represent the same expression.
+
+[.FillVerticalSpace, cols="^.^1a,^.^4a,^.^4a,^.^4a", stripes="none", options="header"]
+|===
+| 	 | Expression in Words				| Circle of Evaluation		| Math Expression
+| @n | 24 increased by 3		 		| @show{(coe '(+ 24 3))}		| @ifsoln{@math{24 + 3}}
+| @n | @math{\frac {1}{3}} less than 4	| @showsoln{(coe '(- 4 1/3))}	| @math{4 - \frac {1}{3}}
+| @n | @ifsoln{the quotient of 24 and 8}| @showsoln{(coe '(/ 24 8))}	| @math{\frac {24}{8}}
+| @n | the product of 9 and 11			| @showsoln{(coe '(* 9 11))}	| @ifsoln{@math{9 * 11}}
+| @n | @ifsoln{16 tripled}				| @show{(coe '(* 16 3))}		| @ifsoln{@math{16 \times 3}}
+| @n | half of 100						| @showsoln{(coe '(* 100 1/2))}	| @ifsoln{@math{\frac {1}{2} \times 100}}
+| @n | the difference between 20 and 8	| @showsoln{(coe '(- 20 8))}	| @ifsoln{@math{20 - 8}}
+| @n | @ifsoln{one-third of 4}			| @showsoln{(coe '(* 1/3 4))}	| @math{\frac{1}{3} \times 4}
+
+|===
+
```

Comment:
Noting that the trickiest translation concept you're introducing is statements using the word "than" and the only examples you've included with these already give the order of the numbers away. Suggest adding at least one more example where students are in charge of getting the order right based on the words.